Blizzard.eai

```diff
@@ -624,6 +638,8 @@ function InitZoom takes nothing returns nothing
         call TriggerRegisterPlayerKeyEventBJ( AdjustZoomUp, Player(i), bj_KEYEVENTTYPE_DEPRESS, bj_KEYEVENTKEY_UP )
         call TriggerRegisterPlayerKeyEventBJ( AdjustZoomDown, Player(i), bj_KEYEVENTTYPE_DEPRESS, bj_KEYEVENTKEY_DOWN )
         set current_zoom[i] = 1650 // Default expected in legacy systems
+        set ZoomKey[i] = true
+        call DisplayTimedTextToPlayer(Player(i),0,0,7,"Zoom Hot Key Switch: |c00d5f038-zoomkey|r")
```

Comment:
```suggestion
        call DisplayTimedTextToPlayer(Player(i),0,0,7,"Enable/Disable Zoom HotKeys: |c00d5f038-zoomkey|r")
```

Reply:
change

Reply:
if add a `Instruction set help button` to Commands.txt , can show all Instruction , maybe good ， they can see any time
but if not use have Dialog‘s mode , it's useless either